cosmotech/data_update_quest_cli/database/redis_file_upload.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) - 2025 - Cosmo Tech
+# This document and all information contained herein is the exclusive property -
+# including all intellectual property rights pertaining thereto - of Cosmo Tech.
+# Any use, reproduction, translation, broadcasting, transmission, distribution,
+# etc., to any person is prohibited unless it has been previously and
+# specifically authorized by written means by Cosmo Tech.
+
+from cosmotech.csm_data.utils.decorators import translate_help
+from cosmotech.orchestrator.utils.translate import T
+
+from cosmotech.data_update_quest.core.database.redis.client import file_upload
```

Comment:
Move the function import to the first line of the command to reduce load charge when commands are detected